### PR TITLE
Translate Romanian Chapter 2

### PR DESCRIPTION
**Pull request by eduard-balamatiuc**
This is the romanian translation of chapter 2.
Ping @lewtun and @stevhliu for a review

### CODE REVIEW

**Review by stevhliu — COMMENTED** [review]
Nice work! The structure and format looks good to me, but you'll need someone to review the accuracy of the translation 🤗

File: chapters/rum/_toctree.yml [hunk 1]
```diff
@@ -26,3 +26,23 @@
   - local: chapter1/10
     title: Quiz de final de capitol
     quiz: 1
+
+- title: 2. Using 🤗 Transformers
```
Comment (grumpycatyo-collab) [inline chapters/rum/_toctree.yml:30]:
```suggestion
- title: 2. Folosirea 🤗 Transformers
```
Just a suggestion since I don't know if titles need to be translated

File: chapters/rum/chapter2/1.mdx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Biblioteca 🤗 Transformers a fost creată pentru a rezolva această problemă. Scopul său este de a oferi un singur API prin care orice model Transformer poate fi încărcat, antrenat și salvat. Principalele caracteristici ale bibliotecii sunt:
+
+- **Simplitate în utilizare**: Descărcarea, încărcarea și utilizarea unui model NLP de ultimă generație pentru inferență pot fi realizate în doar două linii de cod.
+- **Flexibilitate**: În esența lor, toate modelele sunt simple clase PyTorch `nn.Module` sau TensorFlow `tf.keras.Model` și pot fi manipulate ca orice alte modele în framework-urile lor respective de învățare automată (ML).
+- **Simplitate**: Aproape că nu se fac abstractizări în întreaga bibliotecă. "All in one file" este un concept de bază: trecerea înainte a unui model este definită în întregime într-un singur fișier, astfel încât codul în sine să fie ușor de înțeles și de modificat.
+
+Această ultimă caracteristică face 🤗 Transformers destul de diferită de alte biblioteci ML. Modelele nu sunt construite pe module
```
Comment (grumpycatyo-collab) [inline chapters/rum/chapter2/1.mdx:16]:
```suggestion
Această ultimă caracteristică face "🤗 Transformers" destul de diferită de alte biblioteci ML. Modelele nu sunt construite pe module 
```
Maybe wrap the title quotes, 'cause when I was reading it, it was kinda confusing for me. (word "face", in Romanian is "to do")

File: chapters/rum/chapter2/2.mdx [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+
+## Preprocesarea cu un tokenizator[[preprocesarea-cu-un-tokenizator]]
+
+La fel ca alte rețele neuronale, modelele Transformer nu pot procesa direct text brut, astfel încât primul pas al pipeline-ului nostru este de a converti intrările de text în numere pe care modelul le poate înțelege. Pentru a face acest lucru, folosim un *tokenizer*, care va fi responsabil pentru:
+
+- Împărțirea datelor de intrare în cuvinte, subcuvinte sau simboluri (cum ar fi punctuația) care se numesc *tokens*
+- Maparea fiecărui token într-un număr întreg
+- Adăugarea de intrări suplimentare care pot fi utile pentru model
+
+Toată această preprocesare trebuie efectuată exact în același mod ca atunci când modelul a fost preantrenat, așa că mai întâi trebuie să descărcăm aceste informații din [Model Hub] (https://huggingface.co/models). Pentru a face acest lucru, folosim clasa `AutoTokenizer` și metoda sa `from_pretrained()`. Folosind numele checkpoint-ului modelului nostru, aceasta va prelua automat datele asociate cu tokenizer-ul modelului și le va stoca în cache (astfel încât acestea să fie descărcate doar prima dată când executați codul de mai jos).
```
Comment (grumpycatyo-collab) [inline chapters/rum/chapter2/2.mdx:75]:
```suggestion
Toată această preprocesare trebuie efectuată exact în același mod ca atunci când modelul a fost preantrenat, așa că mai întâi trebuie să descărcăm aceste informații din [Model Hub](https://huggingface.co/models). Pentru a face acest lucru, folosim clasa `AutoTokenizer` și metoda sa `from_pretrained()`. Folosind numele checkpoint-ului modelului nostru, aceasta va prelua automat datele asociate cu tokenizer-ul modelului și le va stoca în cache (astfel încât acestea să fie descărcate doar prima dată când executați codul de mai jos).
```

File: chapters/rum/chapter2/2.mdx [hunk 4]
```diff
@@ -0,0 +1,363 @@
...
+    "I've been waiting for a HuggingFace course my whole life.",
+    "I hate this so much!",
+]
+inputs = tokenizer(raw_inputs, padding=True, truncation=True, return_tensors="tf")
+print(inputs)
+```
+{/if}
+
+ 
+Nu vă faceți încă griji cu privire la padding și trunchiere; le vom explica mai târziu. Principalele lucruri de reținut aici sunt că puteți trece o propoziție sau o listă de propoziții, precum și specificarea tipului de tensori pe care doriți să îi primiți înapoi (dacă nu este trecut niciun tip, veți primi o listă de liste ca rezultat).
```
Comment (grumpycatyo-collab) [inline chapters/rum/chapter2/2.mdx:114]: The word `trecut` is confusing? Maybe use `specificat`?
```
“dacă nu este specificat niciun tip, veți primi o listă de liste ca rezultat.”
```

File: chapters/rum/chapter2/3.mdx [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+
+{/if}
+
+În exemplul de cod de mai sus nu am utilizat `BertConfig` și, în loc de acesta, am încărcat un model preantrenat prin intermediul identificatorului `bert-base-cased`. Acesta este un checkpoint al modelului care a fost antrenat chiar de autorii BERT; puteți găsi mai multe detalii despre acesta în [model card](https://huggingface.co/bert-base-cased).
+
+Acest model este acum inițializat cu toate weight-urile din checkpoint. Acesta poate fi utilizat direct pentru inferență pe sarcinile pe care a fost antrenat și poate fi, de asemenea, ajustat pe o sarcină nouă. Prin antrenarea cu weight-uri preantrenate în prealabil, putem obține rapid rezultate bune.
+
+Greutățile au fost descărcate și stocate în cache (astfel încât apelurile viitoare la metoda `from_pretrained()` nu le vor descărca din nou) în folderul cache, care are ca valoare implicită *~/.cache/huggingface/transformers*. Puteți personaliza folderul cache prin setarea variabilei de mediu `HF_HOME`.
```
Comment (grumpycatyo-collab) [inline chapters/rum/chapter2/3.mdx:146]:
```suggestion
Greutățile au fost descărcate și stocate în memorie cache (astfel încât apelurile viitoare la metoda `from_pretrained()` nu le vor descărca din nou) în folderul cache, care are ca valoare implicită *~/.cache/huggingface/transformers*. Puteți personaliza folderul cache prin setarea variabilei de mediu `HF_HOME`.
```
I'd specify. However idk if it's a folder or a type of memory

**Review by grumpycatyo-collab — APPROVED** [review]
Just minor suggestions overall
